src/store/modules/recipeStore.js

```diff
@@ -67,33 +67,32 @@ export const recipeStore = {
 		reset_hasClapped(context) {
 			context.commit('RESET_HASCLAPPED');
 		},
-		post_comment(context, payload) {
-			this.state.api.client.post(
-				`${process.env.BASE_URL}comments`,
-				payload,
-				{
-					headers: {
-						'Authorization': `Bearer ${this.state.access_token}`
-					}
-				}).then(() => {
-					context.commit('COMMENT_POSTED', payload.breadcrumb);
-				}).catch((error) => {
-					context.commit('HANDLE_ERROR', error.response);
-				})
+		async fetch_ingredient_thumbnail(context, ingredient){
```

Comment:
I really donot think this should be async, what is the reasoning behind this?

Reply:
I have moved this to the component